frontend/src/components/results/PlayerResultsItem.tsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import styled from 'styled-components';
+import { Player, Submission } from '../../api/Game';
 import { LowMarginText, Text } from '../core/Text';
 import { Color } from '../../api/Color';
-import { Player } from '../../api/Game';
-import { useBestSubmission } from '../../util/Hook';
+import { useBestSubmission, useGetScore, useGetSubmissionTime } from '../../util/Hook';
 import Language, { displayNameFromLanguage } from '../../api/Language';
 import { TextButton } from '../core/Button';
-import { getScore, getSubmissionCount, getSubmissionTime } from '../../util/Utility';
+// import { getScore, getSubmissionCount, getSubmissionTime } from '../../util/Utility';
```

Comment:
I think I was the one who commented this out and forgot to delete it lol, but could you remove this? Also, could you check if these are still be used (and if so, whether than can be replaced by the new hooks you made) and delete them if not?

Reply:
Actually, looking at the rest of this file, those three methods are duplicated in this component as well, so something to keep in mind - maybe that means the methods can be deleted from this file and you can reuse those util ones, or maybe both can be deleted, etc.